### DISCUSSION TITLE
is it possible to close kitty_os_window inside vim/neovim?

### DISCUSSION THREAD

**Discussion by NEX-S** [kovidgoyal/kitty#5780] [Q&A]
i wanna map `<C-q>` to close current kitty os_window, but I tried `os.exit()` `os.execute('quit')` `vim.fn.system('quit')` it's not work :(

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/remote-control/

**Reply by NEX-S** [reply]
appreciate your reply, sir! have a nice day :smiley_cat:
